File: workspace-servidor/flotix2021/src/main/java/com/flotix/response/bean/ServerResponseRol.java
package com.flotix.response.bean;

import java.util.List;

import com.flotix.dto.RolDTO;

/**
 * Objeto de respuesta con la informacion del rol
 * 
 * @author Flor
 *
 */
public class ServerResponseRol {

	private List<RolDTO> listaRol = null;

	private ErrorBean error = new ErrorBean();

	public ServerResponseRol() {
		super();
	}

	public List<RolDTO> getListaRol() {
		return listaRol;
	}

	public void setListaRol(List<RolDTO> listaRol) {
		this.listaRol = listaRol;
	}

	public ErrorBean getError() {
		return error;
	}

	public void setError(ErrorBean error) {
		this.error = error;
	}
}

File: workspace_presentacion/Flotix2021/Flotix2021/ModelResponse/ServerResponseRol.cs
using Flotix2021.ModelDTO;
using System.Collections.Generic;

namespace Flotix2021.ModelResponse
{
    class ServerResponseRol
    {
        private List<RolDTO> _listaRol;

        public List<RolDTO> listaRol
        {
            get { return _listaRol; }
            set { _listaRol = value; }
        }

        private ErrorBean _error;

        public ErrorBean error
        {
            get { return _error; }
            set { _error = value; }
        }
    }
}
